src/modules/FFmpeg/FFmpeg.cpp

```diff
@@ -249,7 +249,7 @@ ModuleSettingsWidget::ModuleSettingsWidget(Module &module, bool vkVideo, bool dx
     demuxerB->setCheckable(true);
     demuxerB->setChecked(sets().getBool("DemuxerEnabled"));
 
-    reconnectNetworkB = new QCheckBox(tr("Try to automatically reconnect network connections"));
+    reconnectNetworkB = new QCheckBox(tr("Try to automatically reconnect to network"));
```

Comment:
The original words are grammatically correct, but they sound a little bit awkward because the voice “connect” exists twice and they are too close. If you insist, I will take your new amendment.

Reply:
Ok, but to me "Try to automatically reconnect to network" sounds like we want to connect to the network (like system network settings), hmm.